example/gh_traffic.py

```diff
@@ -0,0 +1,21 @@
+"""
+An simple tool to pull GitHub Traffic Statics for a repository
+"""
+
+import json
+import urllib2
+
+auth_token = 'token ' + raw_input("GitHub personal access token: ")
+
+repo_path = raw_input("GitHub repo (username/reponame): ")
```

Comment:
@animeshsingh: Replying to your comment in README.md here.

What if we change this line to this?:

```python
repo_path = raw_input("GitHub repo (username_or_orgname/reponame): ")
```